src/autopas/selectors/tuningStrategy/GaussianModel/GaussianCluster.h

```diff
@@ -0,0 +1,293 @@
+      _evidenceMinVector = std::make_pair(inputDiscrete, inputContinuous);
+      ;
+    } else if (output > _evidenceMaxValue) {
+      _evidenceMaxValue = output;
+      _evidenceMaxVector = std::make_pair(inputDiscrete, inputContinuous);
+      ;
+    }
+
+    _clusters[clusterIdx].addEvidence(inputContinuous, output);
+    ++_numEvidence;
+  }
+
+  /**
+   * Get the evidence with the smallest output value
+   * @return input of min
+   */
+  std::pair<VectorDiscrete, VectorContinuous> getEvidenceMin() {
+    if (_numEvidence == 0) {
+      utils::ExceptionHandler::exception("GaussianCluster has no evidence");
+    }
+
+    return _evidenceMinVector;
+  }
+
+  /**
+   * Get the evidence with the highest output value
+   * @return input of max
+   */
+  std::pair<VectorDiscrete, VectorContinuous> getEvidenceMax() {
+    if (_numEvidence == 0) {
+      utils::ExceptionHandler::exception("GaussianCluster has no evidence");
+    }
+
+    return _evidenceMaxVector;
+  }
+
+  /**
+   * Calculate the acquisition for all possible combinations of discrete tuples and continuous tuples in samples
+   * @param af function to optimize
+   * @param neighbourFun function which generates neighbours of given discrete tuple
+   * @param samples continuous tuples
+   * @return vector of pairs - vectors and corresponding acquisition
+   */
+  std::vector<VectorAcquisition> sampleAcquisition(
+      AcquisitionFunctionOption af, const std::function<std::vector<Eigen::VectorXi>(Eigen::VectorXi)> &neighbourFun,
+      const std::vector<VectorContinuous> &samples) {
+    std::vector<VectorAcquisition> acquisitions;
+
+    for (auto currentContinuous : samples) {
+      // calc means of given continuous tuple for each cluster
+      std::vector<double> means;
+      std::vector<double> vars;
+      std::vector<double> stddevs;
+      means.reserve(_clusters.size());
+      for (const auto &cluster : _clusters) {
+        means.push_back(cluster.predictMean(currentContinuous));
+
+        double var = cluster.predictVar(currentContinuous);
+        vars.push_back(var);
+        stddevs.push_back(std::sqrt(var));
+      }
+
+      // get maximum acquisition considering neighbours
+      VectorDiscrete currentDiscrete = Eigen::VectorXi::Zero(_dimRestriction.size());
+      for (size_t i = 0; i < _clusters.size(); ++i, discreteIncrement(currentDiscrete)) {
+        auto neighbours = neighbourFun(currentDiscrete);
+
+        double mixMean = means[i];
+        double mixVar = vars[i];
+        double weightSum = 1.;
+
+        // sum over neighbours
+        for (const auto &n : neighbours) {
+          size_t n_index = getIndex(n);
+          // ignore clusters without evidence
+          if (_clusters[i].numEvidence() > 0) {
+            // weight based on 2-wasserstein distance
+            double distance = std::pow(means[n_index] - means[i], 2) + std::pow(stddevs[n_index] - stddevs[i], 2);
+            double n_weight = 1. / (1 + distance / vars[i]);
```

Comment:
```suggestion
            double n_weight = vars[i] / (vars[i] + distance);
```
is the same with one less division, isn't it?^^